src/index.html

```diff
@@ -5,8 +5,350 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <nav class="nav header__nav">
+        <a href="#">
+          <img
+           src="images/logo.png"
+           alt="logo"
+           class="nav__homepage"
+          >
+        </a>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link--is-active is-active">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="products" data-qa="card">
```

Comment:
I don't think that we really need two layers of wrappers here.

Reply:
i need it for position this wrapper in center of screen and make max. width. flex-wrap don`t work without this. 
I don`t know how can i realized this without wrapper